collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/DeleteProjectTest.kt

```diff
@@ -41,4 +42,32 @@ class DeleteProjectTest {
             .clickAdminSettings()
             .deleteLastProject()
     }
+
+    @Test
+    fun deleteProjectWithInstancesTest() {
```

Comment:
For a test like this I think we can have a name that references the precondition and the result of that. An example could be `whenProjectHasUnsentInstances_cannotDeleteProject` or `whenProjectHasUnsentInstances_clickingDelete_showsMessage` or even just `cantDeleteProjectWithUnsentInstances`.